### PR TITLE
Bug Report Template: Warn against sharing private material on public forum

### PR DESCRIPTION
**Pull request by KevinTCoughlin**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

Adds warning text to the "Bug Report" template suggesting not to share that which you do not want exposed to the public internet in your bug report (Github issue).

Defer actual text to wordsmiths @micahgodbolt @ecraig12345 @cliffkoh 

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/8325)

### CODE REVIEW

**Comment by KevinTCoughlin** [comment]
Apparently my IDE has an opinionated Markdown linter from my content writing work. I'll investigate and revise the diff.

**Comment by ecraig12345** [comment]
That's a very enthusiastic linter! I actually wonder if the wording should be a bit stronger.

**Comment by KevinTCoughlin** [comment]
> That's a very enthusiastic linter! I actually wonder if the wording should be a bit stronger.

I was also unsure where the line should be positioned. I'm open to any and all suggestions.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,50 +1,52 @@
...
+  * Goto https://aka.ms/fabricpen for a starting codepen
   * You can also use the "Export to Codepen" feature for the various components in our documentation site.
-  * Alternatively, you can also use https://aka.ms/fabricdemo to get permanent repro links if the repro occurs with an example. 
+  * Alternatively, you can also use https://aka.ms/fabricdemo to get permanent repro links if the repro occurs with an example.
     (A permanent link is preferable to "use the website" as the website can change)
- 
+
 Note that if you do not provide enough information to reproduce the issue, we may not be able to take action on your report.
+
+Also note that this Github repository is a public forum.
```
Comment (ecraig12345) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Maybe move to the beginning and say something like:

> WARNING: This GitHub repository is a public forum. DO NOT share links or info which should not be available on the public internet: i.e. internal issue tracking links, screenshots of products in development, etc.

But as you pointed out elsewhere, they can still ignore it regardless of all our efforts...

Comment (ecraig12345) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Or closer to the beginning at least.

Comment (micahgodbolt) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Links are typically ok, assuming the URL doesn't reveal anything innapropriate.

**Comment by KevinTCoughlin** [comment]
Will re-open

**Pull request closed without merging**